cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -154,300 +130,3 @@ def store_flow_cell_data(self, parsed_flow_cell: FlowCellDirectoryData) -> None:
             flow_cell_run_dir=self.demux_api.run_dir,
             store=self.status_db,
         )
-
-
-class DemuxPostProcessingHiseqXAPI(DemuxPostProcessingAPI):
```

Comment:
The new demux post processing flow should handle any kind of input when demultiplexing. (i.e. bcl2fastq and bcl convert structures). This API would be obsolete.